docs/ARCHITECTURE.rst

```diff
@@ -937,3 +937,50 @@ Algorithms
 The algorithms for each listing type can be found in the open-source
 `scality/Arsenal <https://github.com/scality/Arsenal>`__ repository, in
 `lib/algos/list <https://github.com/scality/Arsenal/tree/master/lib/algos/list>`__.
+
+Encryption
+===========
+
+About Server Side Encryption CloudServer has its own bucket level encryption.
+
+Differentiations
+------------------
+
+With the real AWS S3, to enable encryption, you have to specify on the object
+level a ``x-amz-server-side-encryption: AES256`` header when you upload an object
+
+With your Scality CloudServer, you have to specify a
+``x-amz-scal-server-side-encryption: AES256`` header on the bucket level
+(ie, when you create your **bucket**).
+This enables encryption for all the objects stored in your Scality CloudServer **bucket**.
+
+Example:
+--------
+
+Creating encrypted bucket named: encryptedbucket
+
+.. code:: shell
+
+    s3cmd mb s3://encryptedbucket --add-header x-amz-scal-server-side-encryption:AES256
```

Comment:
This won't work.  There is no ability to add a header on a bucket put in the regular clients since this is a divergence from the regular S3 spec.  instead, point them to our create encrypted bucket tool in the bin directory.